src/components/StarRating/StarRating.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import ReactStars from "react-rating-stars-component";
+import './StarRating.scss';
+
+type StarRatingProps = {
+  averageRating: number
+}
+
+function StarRating({ averageRating }: StarRatingProps) {
+  return (
+    <div className="StarRating">
+      <ReactStars 
+      count={5}
+      size={20}
+      value={averageRating}
+      isHalf = {true}
```

Comment:
funktioniert das wenn das so auseinander ist?

Reply:
yes, it works. Another question I cannot answer - why the files structure here is so weird))) in my source code in VS it is different (i mean models vs Models)